primitives/session-keys/src/vrf.rs

```diff
@@ -34,6 +34,18 @@ impl From<NimbusId> for VrfId {
 	}
 }
 
+pub fn nimbus_id_from_bytes(bytes: &[u8]) -> NimbusId {
+	// let sr25519_as_bytes: [u8; 32] = bytes.into();
+	// sr25519::Public::unchecked_from(sr25519_as_bytes).into()
+	todo!()
+}
```

Comment:
need to know how to convert &[8] to [u8; 32] 
looking at https://stackoverflow.com/questions/58478116/how-to-convert-u8-into-u8-64

Reply:
@4meta5 I think it's better to use Decode directly, with something like that:

```rust
let keys: (NimbusId, T::Keys) = bytes.decode().map_err(…)?;
```